perf-tool/src/agentOptions.cpp

```diff
@@ -57,11 +57,17 @@ void invalidRate(const std::string& function, const std::string& command, int sa
     printf("Invalid rate with parameters: {functionality: %s, command: %s, sampleRate: %i}\n", function.c_str(), command.c_str(), sampleRate);
 }
 
-void modifyMonitorEvents(const std::string& function, const std::string& command, int rate)
+void invalidStackTraceDepth(const std::string& function, const std::string& command, int stackTraceDepth)
+{
+    printf("Invalid stackTraceDepth with parameters: {functionality: %s, command: %s, stackTraceDepth: %i}\n", function.c_str(), command.c_str(), stackTraceDepth);
```

Comment:
Given that this project uses better C++ than OJ9 (eg `std::string`, etc), shouldn't we just use `cout`?

Reply:
Comment https://github.com/eclipse/openj9-utils/pull/7#pullrequestreview-542464211 from Igor suggested to use printf. Currently all the code uses printf, much like openJ9 project does. from Igor suggested to use printf. Currently all the code uses printf, much like openJ9 project does.